File: Sequence.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>
#include <string.h>

void Sequential(char n[], char letter)
{
    bool is_found = false;
    int i = 0;
    while (!is_found && i < strlen(n))
    {
        if (n[i] == letter)
        {
            is_found = true;
        }
        else
        {
            i++;
        }
    }
    if (is_found)
    {
        printf("Letter found!\n%c", letter);
    }
}

int main()
{
    char pool[] = "hello";
    Sequential(pool, 'o');
    return 0;
}

File: Sequence.java
import java.lang.*;

public class Sequence {
    public static void main(String[] args) {
        String[] word = { "h", "e", "l", "l", "o" };
        String find = "o";
        Sequential(word, find);
    }

    public static void Sequential(String[] n, String letter) {
        boolean isFound = false;
        int i = 0;
        while (!isFound && i < n.length) {
            if (n[i] == letter) {
                isFound = true;
            } else {
                i++;
            }
        }
        if (isFound) {
            System.out.println("Letter found:" + letter);
        }
    }
}